### DISCUSSION TITLE
/java8/stream.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#68] [Announcements]
# /java8/stream.html

Java程序员进阶之路，小白的零基础Java教程，从入门到进阶，Java 8 Stream流详细用法

https://tobebetterjavaer.com/java8/stream.html

**Comment by echo0109** [comment]
`noneMatch()，只要有一个元素匹配传入的条件，就返回 false；如果全部匹配，则返回 true。`
写错了，后面那句：如果全部不匹配，则返回true

**Comment by Hehebulu** [comment]
《noneMatch()，只要有一个元素匹配传入的条件，就返回 false；如果全部匹配，则返回 true。》
后面那句应该是写错了：如果全不匹配，则返回true
